vllm/model_executor/layers/quantization/aqlm.py

```diff
@@ -0,0 +1,233 @@
+# Supports AQLM compression, see https://github.com/Vahe1994/AQLM and https://arxiv.org/pdf/2401.06118.pdf
+
```

Comment:
This is my big contribution to the project, ie, not code cribbed from AQLM github.  Do we have a comment we put in here to indicate it's ours or something?

Reply:
@mgoin ? @robertgshaw2-neuralmagic

Reply:
I don't think we should attribute ourselves here since this code was written for vLLM and thus is apache 2.0 under their license

Reply:
okay.